.travis.yml

```diff
@@ -49,6 +49,12 @@ jobs:
         - yarn dev &
         - yarn wait-on "http-get://localhost:3000" && yarn cypress run
         - kill $(jobs -p) || true
+    - stage: longer tests
+      name: "Partners site Cypress tests"
+      script:
+        - yarn cypress install
+        - yarn test:app:partners:headless
+        - kill $(jobs -p) || true
```

Comment:
I think the reason it's hitting the watch limit is because it's doing a `yarn dev` on the backend, which has `--watch`. Try doing the same thing as the public tests above, where we do the commands separately and not `dev` in the backend.